### PR TITLE
add warnings for consistency check

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <hustmh@gmail.com>

### What is changed, added or deleted? (Required)
Currently, consistency-check isn't compatible with TiDB GC (see https://github.com/tikv/tikv/issues/12564), add warnings for related docs about configuration of consistency-check to prevent users enable it.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hicqu — APPROVED** [review]

**Review by BornChanger — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -711,12 +711,6 @@ raftstore 相关的配置项。
 + 最小值：0
 + 单位：MB
 
-### `consistency-check-interval`
```
Comment (TomShawn) [inline tikv-configuration-file.md:714]: 建议加上一个警告说明：

> **警告：**
>
> 该配置项存在已知问题，继续使用可能会存在风险，建议不设置该配置项。

File: dynamic-config.md [hunk 2]
```diff
@@ -147,7 +147,6 @@ show warnings;
 | raftstore.max-leader-missing-duration | 允许副本处于无主状态的最长时间，超过将会向 PD 校验自己是否已经被删除 |
 | raftstore.abnormal-leader-missing-duration | 允许副本处于无主状态的时间，超过将视为异常，标记在 metrics 和日志中 |
 | raftstore.peer-stale-state-check-interval | 触发检验副本是否处于无主状态的时间间隔 |
-| raftstore.consistency-check-interval | 触发一致性检查的时间间隔 |
```
Comment (TomShawn) [inline dynamic-config.md:150]: ditto，加说明即可

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -716,6 +716,7 @@ raftstore 相关的配置项。
 + 触发一致性检查的时间间隔，0 表示不启用。
```
Comment (TomShawn) [inline tikv-configuration-file.md:716]:
```suggestion

> **警告：**
>
> 开启一致性检查对集群性能有影响，并且和 TiDB GC 操作不兼容，不建议在生产环境中使用。

+ 触发一致性检查的时间间隔，0 表示不启用。
```

File: dynamic-config.md [hunk 4]
```diff
@@ -147,7 +147,7 @@ show warnings;
...
+| raftstore.consistency-check-interval | 触发一致性检查的时间间隔 (目前与 TiDB GC 操作不兼容) |
```
Comment (TomShawn) [inline dynamic-config.md:150]:
```suggestion
| raftstore.consistency-check-interval | 触发一致性检查的时间间隔（不建议使用该配置项，因为与 TiDB GC 操作不兼容）|
```

File: tikv-control.md [hunk 5]
```diff
@@ -375,6 +375,7 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
 >
 > - **该命令只支持远程模式**。
 > - 即使该命令返回了成功信息，也需要检查是否有 TiKV panic 了。因为该命令只是向 Leader 请求进行一致性检查，但整个检查流程是否成功并不能在客户端知道。
+> - 目前与 TiDB GC 操作不兼容，存在一定的误报可能。
```
Comment (TomShawn) [inline tikv-control.md:378]:
```suggestion
> - 目前与 TiDB GC 操作不兼容，存在一定的误报可能，因此不建议使用。
> - **该命令只支持远程模式**。
> - 即使该命令返回了成功信息，也需要检查是否有 TiKV panic 了。因为该命令只是向 Leader 请求进行一致性检查，但整个检查流程是否成功并不能在客户端知道。
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -716,6 +716,7 @@ raftstore 相关的配置项。
...
 + 默认值：0s
 + 最小值：0
++ 注意：开启后对集群性能有影响，并且和目前版本的 TiDB GC 操作不兼容，不建议在生存环境中使用
```
Comment (TomShawn) [inline tikv-configuration-file.md:719]:
```suggestion
```

File: tikv-control.md [hunk 7]
```diff
@@ -373,6 +373,7 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
 
 > **注意：**
 >
+> - 目前 consistency-check 与 TiDB GC 操作不兼容，存在误报的可能，因此不建议使用该命令。
```
Comment (TomShawn) [inline tikv-control.md:376]:
```suggestion
> - 目前 consistency-check 与 TiDB GC 操作不兼容，存在误报错误的可能，因此不建议使用该命令。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**